### ISSUE TITLE
RfE: Allow to render a book doctype as html

### ISSUE THREAD

**Issue by pilhuhn** [asciidoctor/asciidoctor#317]
Currently the html doctype assumes one = (h1) as document header only and starts all the sections at == (h2)

The book doctype allowes multiple = (h1) in a document, but -b docbook renders xml, which needs further processing to be viewable as html

While having multiple h1 in a html document is probably not completely correct, allowing to render such book doctypes as html directly from within asciidoctor would allow for a quick preview. Also books created like this could be rendered/viewed as html on github, while the document structure would be suitable for a "real book".

So basically what I am saying is to allow for asciidoctor to just run `asciidoctor input.adoc` to create html, but honor an internal `:doctype: book` to use the book processing rules (which would normally apply for the `-b dockbook45` call.
